packages/vue/src/tracing.ts

```diff
@@ -66,7 +66,9 @@ export const createTracingMixins = (options: TracingOptions): Mixins => {
       continue;
     }
 
-    for (const internalHook of internalHooks) {
+    internalHooks.forEach((internalHook, hookType) => {
+      const isBeforeHook = hookType === 0;
```

Comment:
Any reason this need to be a variable? Only used once in the if statement - and I think the meaning is pretty clear without the need of a variable.

Reply:
ok let me update it

Reply:
updated